sleap_nn/data/custom_datasets.py

```
@@ -1,6 +1,7 @@
 """Custom `torch.utils.data.Dataset`s for different model types."""
 
 from kornia.geometry.transform import crop_and_resize
+from itertools import cycle
```

Ruff (0.8.2)

`itertools.cycle` imported but unused

Remove unused import: `itertools.cycle`

(F401)